tests/unit/test_mock_server.py

```diff
@@ -173,7 +173,7 @@ def test_register_handlers_registers_noop_handler_for_message_with_no_ack(
     )
     server = new_mock_asynction_socket_io(spec)
 
-    server._register_handlers()
+    server._register_handlers([])
```

Comment:
No need for these little changes if we initialise the new argument with an empty tuple. Have also commented this above.